pkg/config/system_probe.go

```diff
@@ -224,6 +224,7 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnv(join(smNS, "max_http_stats_buffered"))
 	cfg.BindEnvAndSetDefault(join(smNS, "max_kafka_stats_buffered"), 100000)
 	cfg.BindEnv(join(smNS, "max_concurrent_requests"))
+	cfg.BindEnvAndSetDefault(join(smNS, "enable_quantization"), false)
```

Comment:
Let's use `AndSetDefault`
the default for boolean is false anyway, but "setting" it, will make it harder to know if a customer explicitly disable it or not